stable/sumokube/templates/daemonset.yaml

```diff
@@ -0,0 +1,64 @@
+{{- if and .Values.sumologic.accessId .Values.sumologic.accessKey -}}
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: {{ .Release.Name }}
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+spec:
+  template:
+    metadata:
+      name: {{ .Release.Name }}
```

Comment:
Can you add the suffix of the app/chart to the pod name spec? Otherwise its hard to tell what pods they are as they are just named after the release. 

Other charts use a "fullname" template for these identifiers:
https://github.com/kubernetes/charts/blob/master/stable/mariadb/templates/deployment.yaml
https://github.com/kubernetes/charts/blob/master/stable/mariadb/templates/_helpers.tpl